plugins/org.eclipse.sirius/src/org/eclipse/sirius/business/internal/image/ImageManagerForWorkspaceResource.java

```diff
@@ -89,7 +91,7 @@ public CreateImageFileProvider getCreateFileFunc(EObject contextObject, String s
         // Create an image file in the images folder of the project
         String platformString = contextObject.eResource().getURI().toPlatformString(true);
         IFile airdFile = ResourcesPlugin.getWorkspace().getRoot().getFile(new Path(platformString));
-        IFolder imageFolder = airdFile.getProject().getFolder(ImageManager.IMAGE_FOLDER_NAME);
+        IFolder imageFolder = airdFile.getProject().getFolder(getImagesFolderName(contextObject));
```

Comment:
Currently, this method can return null. 
Change it so that it returns ImageManager.IMAGE_FOLDER_NAME as a fall back

Reply:
Done